addons/website/static/src/components/dialog/page_properties.xml

```diff
@@ -61,20 +61,20 @@
 
 <t t-name="website.DeletePageDialog">
     <WebsiteDialog title="title" close="props.close">
-        <p t-if="props.resIds.length > 1">Are you sure you want to delete those pages?</p>
-        <p t-else="">Are you sure you want to delete this page?</p>
-        <p class="text-warning">Don't forget to update all links referring to it.</p>
+        <p t-if="props.resIds.length > 1">Ready to make those web pages disappear into thin air? Poof it will be gone forever!</p>
+        <p t-else="">Ready to make this web page disappear into thin air? Poof it will be gone forever!</p>
+        <p class="text-warning">But before deleting it, make sure you update all links referring to it. That way, your customers won't bump into the not-so-famous 404 error page..</p>
```

Comment:
Weren't those messages supposed to be only for "rare" error cases that you should not encounter with basic flow?
Here, you will face this message pretty much everytime you delete a page IMO, pages URL are often cross referenced, or in the website menu.
I think the change in line 66 was enough to bring a touch of "fun" while not being too invasive. But I am fine to keep the whole change if you prefer, it's fine.